src/containers/Chat/ChatConversations/ChatConversations.test.tsx

```diff
@@ -9,6 +9,17 @@ import { MockedProvider } from '@apollo/client/testing';
 import { ChatConversationMocks } from './ChatConversations.test.helper';
 
 const cache = new InMemoryCache({ addTypename: false });
+jest.mock('react-i18next', () => ({
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (str: string) => str,
+      i18n: {
+        changeLanguage: () => new Promise(() => {}),
+      },
+    };
+  },
+}));
```

Comment:
Add this in setupTests.ts file so that this mock can be used in all the test files.

Reply:
Currently App test failing, when we add i18n mock in setupTest.js.